tester.py

```diff
@@ -72,6 +72,22 @@ def dnanexus_cli(url, filename, reference_name=None, start=None, end=None):
             subprocess.check_call(cmd, stdout=outfile)
 
 
+def sanger_cli(url, filename, reference_name=None, start=None, end=None):
+                                   
```

Comment:
Is this client publicly available? If so, could you put the URL into the comments for this function? This would be useful for context.